src/frontend-scripts/components/section-main/Profile.jsx

```diff
@@ -106,6 +106,27 @@ class ProfileWrapper extends React.Component {
 		);
 	}
 
+  currentGameFnctn(gameId) {
+    window.location = `#/table/${gameId}`;
+  }
+
+	currentGame() {
+
+    if (userInfo.username || gameInfo.publicPlayersState.find(player => player.username === userInfo.userName)) {

```

Comment:
Are you sure `gameInfo` is defined? Not sure where that's coming from.

Reply:
Thank you for that. I forgot to define `gameInfo`.